repository/MonticelloFileTree-Core.package/MCFileTreeRepository.class/instance/readerClassForFileNamed..st

```diff
@@ -0,0 +1,4 @@
+actions
+readerClassForFileNamed: fileName
+	^ "MCReader readerClassForFileNamed: fileName"
+		MCFileTreeReader
```

Comment:
Ah oops. Indeed, that should be a redefinition.

On 6 March 2015 at 20:19, Dale Henrichs <notifications@github.com> wrote:

> In
> repository/MonticelloFileTree-Core.package/MCFileTreeRepository.class/instance/readerClassForFileNamed..st
> <https://github.com/dalehenrich/filetree/pull/151#discussion_r25969587>:
>
> > @@ -0,0 +1,4 @@
> > +actions
> > +readerClassForFileNamed: fileName
> > +	^ "MCReader readerClassForFileNamed: fileName"
> > +		MCFileTreeReader
>
> But this was a change to MCFileTreeRepository (which is why I called it
> out) ... agree with using cypress:// since the format is completely
> different ... I will have to teach Metacello to recognize this, but that's
> no big deal ... also cypress:// means that it will be easier to
> eventually break away from filetree:// (in the distant future:)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dalehenrich/filetree/pull/151/files#r25969587>.
>



-- 
Damien Pollet
type less, do more [ | ] http://people.untyped.org/damien.pollet